### DISCUSSION TITLE
Is there any reason for initial delay being more 15minutes before starting polling Isolation group queue?

### DISCUSSION THREAD

**Discussion by vamsibandarupalli** [Netflix/conductor#2840] [Q&A] [answered]
There is initialDelay of 1000 seconds for IsolationGroup Queue to start polling, is there any specific reason to keep this initial delay?

Link: https://github.com/Netflix/conductor/blob/c70a0376bc29e444b3ad0a75a71320a1313911e3/core/src/main/java/com/netflix/conductor/core/execution/tasks/IsolatedTaskQueueProducer.java#L73

**Comment by aravindanr** [comment, marked as answer]
That is a bug. Will push out a fix in the next release.

**Reply by aravindanr** [reply]
https://github.com/Netflix/conductor/pull/2848
